lib/ci-stack.ts

```diff
@@ -53,6 +60,8 @@ export interface CIStackProps extends StackProps {
   readonly enableViews?: boolean;
   /** Use Production Agents */
   readonly useProdAgents?: boolean;
+  /** Specify jenkins instance type */
+  readonly jenkinsInstanceType?: string;
```

Comment:
Let's keep it constant across the code. Either deploymentType or JenkinsInstanceType. 
Also maybe the type should be `string | DeploymentType`